src/Compilers/VisualBasic/Portable/Syntax/SyntaxFacts.vb

```diff
@@ -107,6 +107,8 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                         Else
                             Return parentQualName.Right Is node
                         End If
+                    Case SyntaxKind.TupleElement
+                        Return True ' all non-token children are types
```

Comment:
Seem I forgot to push my latest change. This line will check the parent and also I will have a test directly on semantic model API.